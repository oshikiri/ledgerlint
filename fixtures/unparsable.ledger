; comment 1
; comment 2
XXX 2022/01/01 XXX
