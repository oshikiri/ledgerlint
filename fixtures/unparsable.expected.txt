fixtures/unparsable.ledger:1 This line is neither comment nor header nor posting
